src/testUtils.tsx

```diff
@@ -30,56 +35,34 @@ interface ExtendedRenderOptions extends Omit<RenderOptions, 'queries'> {
   queryClient?: QueryClient;
 }
 
-export function renderComponentWithBrowserRouter(
+export function renderComponentWithRouterProvider(
   ui: React.ReactElement,
+  urlPathKey?: keyof typeof paths,
+  pathName?: string,
```

Comment:
```suggestion
  initialEntry?: string,
```
I think this makes it a bit clearer.